

Since DevFS does not seem to be actively maintained anymore we might
need to push it a bit in the future ...


  - Rene Rebe <rene@exactcode.de>

--- devfsd/compat_name.c.orig	2002-01-25 08:06:35.000000000 +0100
+++ devfsd/compat_name.c	2003-12-07 00:37:49.000000000 +0100
@@ -3,6 +3,7 @@
     Compatibility name file for  devfsd  (build compatibility names).
 
     Copyright (C) 1998-2002  Richard Gooch
+                  2003       Rene Rebe <rene@exactcode.de>
 
     This program is free software; you can redistribute it and/or modify
     it under the terms of the GNU General Public License as published by
@@ -58,9 +59,11 @@
   new compatibility names for IDE devices. Consolidated SCSI code. Consolidated
   IDE code.
 
-    Last updated by Richard Gooch   25-JAN-2002: Added compatibility entries
+    Updated by      Richard Gooch   25-JAN-2002: Added compatibility entries
   for parallel port generic ATAPI interface.
 
+    Last updated by Rene Rebe       07-DEZ-2003: Added compatibility entires
+  for the usb/ directory.
 
 */
 #include <linux/major.h>
@@ -126,6 +129,7 @@
     {"tts/",       "ttyS%s"},       /*  Generic serial: must be after others */
     {"cua/",       "cua%s"},        /*  Generic serial: must be after others */
     {"input/js",   "js%s"},         /*  Joystick driver                      */
+    {"usb/",       "usb%s"},        /*  USB devices like lp and scanner      */
     {NULL,         NULL}
 };
 
--- devfsd/modules.devfs.orig	2002-03-25 01:00:33.000000000 +0100
+++ devfsd/modules.devfs	2003-12-07 00:37:20.000000000 +0100
@@ -1,5 +1,6 @@
 # /etc/modules.devfs
 # Richard Gooch  <rgooch@atnf.csiro.au>		24-MAR-2002
+# Rene Rebe <rene@exactcode.de>		07-DEZ-2003
 #
 # THIS IS AN AUTOMATICALLY GENERATED FILE. DO NOT EDIT!!!
 # THIS FILE WILL BE OVERWRITTEN EACH TIME YOU INSTALL DEVFSD!!!
@@ -128,6 +129,14 @@
 # Raw I/O devices
 alias	/dev/rawctl		/dev/raw
 
+# USB
+alias    /dev/usb/scanner	scanner
+alias    /dev/usb/scanner*	/dev/usb/scanner
+alias    /dev/usbscanner*	/dev/usb/scanner
+
+alias    /dev/usb/lp		printer
+alias    /dev/usb/lp*		/dev/usb/lp
+alias    /dev/usblp*		/dev/usb/lp
 
 # Pull in the configuration file. Do this last because modprobe(8) processes in
 # per^H^H^Hreverse order and the sysadmin may want to over-ride what is in the
